README.md

```diff
@@ -334,17 +553,102 @@ Now deploy using make:
 
 If successful, you should be able to see the Swagger API documentation at:
 
-    https://<domain_name>
+    https://${API_DOMAIN_NAME}
 
 And you should be able to list bundles like this:
 
-    curl -X GET "https://<domain_name>/v1/bundles" -H  "accept: application/json"
+    curl -X GET "https://${API_DOMAIN_NAME}/v1/bundles" -H  "accept: application/json"
 
 #### Monitoring
 
 Please see the [data-store-monitor](https://www.github.com/humancellatlas/data-store-monitor) repo for additional
 monitoring tools.
 
+#### Updating Environment Variables
+
+Updating environment variables defined in either `environment` or `environment.local` requires
+some care when generating Terraform files from those environment variables.
+
+Many environment variables make their way into Terraform files via templates that are made with
+make commands, so when you change your `environment` or `environment.local` file, it is not enough
+to just run `source environment`.
+
+The Terraform files that store variable values are called `variables.tf` and are located in
+subdirectories of the `infra/` folder. These `variable.tf` files are Terraform files automatically
+generated by the `infra/build_deploy_config.py` script. To remake these files:
+
+```
+# Delete all variables.tf files in infra/
```

Comment:
you dont actually need to delete them, they get overwritten the make commands are used in `/infra`

Reply:
it should be noted that the make commands should always be used, otherwise it leads to operators running into problems with deployment.

Reply:
fixed by 9549335